pkg/restapi/rp/controller_test.go

```diff
@@ -68,3 +70,27 @@ func (m *mockAriesStorageProvider) TransientStorageProvider() ariesstorage.Provi
 
 	return ariesmockstorage.NewMockStoreProvider()
 }
+
+type stubPresentProofClient struct {
```

Comment:
We can move this to common [`mock`](https://github.com/trustbloc/edge-adapter/tree/master/pkg/internal/mock/) folder.

Reply:
@rolsonquadras done